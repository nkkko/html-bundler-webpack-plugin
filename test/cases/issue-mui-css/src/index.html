<!DOCTYPE html>
<html>
  <head>
    <title>Test</title>
    <link href="./style.css" rel="stylesheet" />
  </head>
  <body>
    <h1>Hello World!</h1>
    <div id="app"></div>
    <script src="index.tsx"></script>
  </body>
</html>