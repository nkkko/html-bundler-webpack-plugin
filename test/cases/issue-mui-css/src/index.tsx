//import { createRoot } from 'react-dom/client';

// TODO: fix - both together not works
import Stack from '@mui/material/Stack';
//import './style.css';

console.log(Stack);

// const root = createRoot(document.getElementById('app')!);
//
// root.render(
//   <Stack>
//     <p>Hello!</p>
//     <p>How are you?</p>
//   </Stack>
// );
