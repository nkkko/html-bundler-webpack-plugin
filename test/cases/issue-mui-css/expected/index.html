<!DOCTYPE html>
<html>
  <head>
    <title>Test</title>
  <link href="index.dc4ea4af.css" rel="stylesheet">
</head>
  <body>
    <h1>Hello World!</h1>
    <div id="app"></div>
    <script src="index.6ed8e15f.js"></script>
  </body>
</html>